docusaurus/docs/protocol/primitives/probabilistic_proofs.md

```diff
@@ -1,23 +1,514 @@
+    ISPPR{"Is P(Gov.ProofRequestProbability) == 1 ? <br> (omitting rand seed details)"}
+    %% Is Proof Available
+    ISPA{"Is Proof Available?"}
+
+    SC --> ISCAT
+
+    ISCAT --> |No| ISPPR
+    ISCAT --> |Yes| PR
+
+    ISPPR --> |No| NP
+    ISPPR --> |Yes| ISPA
+
+    ISPA --> |"Yes<br>(Assume Proof is valid)"| DR
+    ISPA --> |No| SLASH
+
+    PR --> ISPA
+    NP --> DR
+
+    style DR fill:#c2f0c2,stroke:#27ae60,stroke-width:2px,color:#000
+    style SLASH fill:#f2a3a3,stroke:#e74c3c,stroke-width:2px,color:#000
+    style NP fill:#f0f0c2,stroke:#f39c12,stroke-width:2px,color:#000
+    style PR fill:#c2d1f0,stroke:#2980b9,stroke-width:2px,color:#000
+
+    classDef decision fill:#B39CD0,stroke:#5D3FD3,color:#000;
+    class ISCAT,ISPPR,ISPA decision;
+```
+
+## Key Question
+
+**What onchain protocol governance parameters need to be selected or created to**
+**deter a Supplier from submitting a false Claim? How can this be modeled?**
+
+## Guarantees & Expected Values
+
+Pocket Network's tokenomics do not provide a 100% guarantee against gaming the system.
+Instead, there's a tradeoff between the network's security guarantees and factors
+like scalability, cost, user experience, and acceptable gamability.
+
+Our goal is to:
+
+- Model the expected value (EV) of both honest and dishonest Suppliers.
+- Adjust protocol parameters to ensure that the expected profit for dishonest behavior is less than that of honest behavior.
+
+A Supplier's balance can change in the following ways:
+
+1. ✅ **Earn rewards for valid Claims with Proofs** (Proof required).
+2. ✅ **Earn rewards for valid Claims without Proofs** (Proof not required).
+3. 🚨 **Earn rewards for invalid Claims without Proofs** (Proof not required).
+4. ❌ **Get slashed for invalid Proofs** (Proof required but invalid).
+5. ❌ **Get slashed for missing Proofs** (Proof required but not provided).
+
+The goal of Probabilistic Proofs is to minimize the profitability of scenario (3🚨)
+by adjusting protocol parameters such that dishonest Suppliers have a negative expected
+value compared to honest Suppliers.
+
+## Modeling an Attack
+
+### Defining a Single (Bernoulli) Trial
+
+We use a [Bernoulli distribution](https://en.wikipedia.org/wiki/Bernoulli_distribution)
+to model the probability of a dishonest Supplier getting caught when submitting false Claims.
+
+- **Trial Definition**: Each attempt by a Supplier to submit a Claim without being required to provide a Proof.
+- **Success**:
+  - A dishonest Supplier gets caught (i.e. is required to provide a Proof and fails, resulting in a penalty)
+  - Taken from the network's perspective
+- **Failure**:
+  - A dishonest Supplier does not get caught (i.e. is not required to provide a Proof and receives rewards without providing actual service)
+  - An honest Supplier is rewarded
+  - All other outcomes
+  - Does not include _short-circuited_ (i.e. Claim.ComputeUnits > ProofRequirementThreshold)
+
+### Onchain Governance Parameters
```

Comment:
I'm wondering if there's a more appropriate/accurate title for this section. It seems to me that these are more like conceptual-level parameters. :thinking: 

```suggestion
### On-chain Governance Parameters
```